### PR TITLE
update civicpy cache only when querying civic

### LINKED ISSUE #21: update civicpy cache only when querying civic
### Description of the bug

the civicpy cache gets updated also when only querying cgi.
The update function lies open in the `query_civic.py` script. As this gets imported by the main script, the update function is triggered even when not needed.
This is inefficient and might lead to downstream issues at the `variantMTB`.

### Command used and terminal output

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by HomoPolyethylen**
Fixes #21

**PR Checklist**

- [x] This comment contains a description of changes (with reason)
- [x] Referenced issue is linked
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.rst` is updated

**Description of changes**

moved `civic.load_cache()`, which updated the civicpy cache into the `query_civic()` function.
Thus it is not triggered through an import (as before) and hence only if civic is actually queried (and not when cgi is queried).
Also added a logger info.

**Technical details**

**Additional context**

### CODE REVIEW

File: querynator/report_scripts/create_report.py [hunk 1]
```diff
@@ -512,9 +512,11 @@ def create_evidence_table(row, width_dict):
...
             exploded_row = pd.DataFrame(
                 data=[
-                    pd.Series(row[col].split(","))
-                    if col not in ["Description", "Source"]
-                    else pd.Series(row[col].split("|"))
+                    (
+                        pd.Series(row[col].split(","))
+                        if col not in ["Description", "Source"]
+                        else pd.Series(row[col].split("|"))
+                    )
```
Comment (mapo9) [inline querynator/report_scripts/create_report.py:519]: what does this change do?

**Review by mapo9 — COMMENTED** [review]
how does this fix the wildtype problem?

File: querynator/report_scripts/create_report.py (hunk 1 above)
Comment (HomoPolyethylen) [reply querynator/report_scripts/create_report.py:519]: this was pythonBlack's doing. gave it a test run and did not seem to have broken anything

**Comment by HomoPolyethylen** [comment]
> how does this fix the wildtype problem?

my bad. it does not. I had that in a separate PR, which I aborted. will open a new one. This PR will only prevent the civicpy cache to be updated when using cgi.
It is a small PR but I thought the issues are not related enough to be combined.
I could also add the wildtype fix here?

**Comment by mapo9** [comment]
> > how does this fix the wildtype problem?
> 
> my bad. it does not. I had that in a separate PR, which I aborted. will open a new one. This PR will only prevent the civicpy cache to be updated when using cgi. It is a small PR but I thought the issues are not related enough to be combined. I could also add the wildtype fix here?

I will merge this one now to dev, then you can create the new one from the update dev

**Pull request merged**